src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+dataSource.user=postgres
```

Comment:
I belive we should not add to pull request such files, and save them in 'DO NOT COMMIT!!!!' changelist